src/StructureElementConnectivityRestraint.cpp

```diff
@@ -65,15 +109,15 @@ float StructureElementConnectivityRestraint::get_model_distance() const {
   
   std::string chain_1 = threading::StructureElement(get_model(), a_).get_chain();
   std::string chain_2 = threading::StructureElement(get_model(), b_).get_chain();  
-  
+  //std::cout << "chain 1: " << chain_1 << " & chain 2: " << chain_2 << std::endl;
   // Compute and return the distance
   if (chain_1 == chain_2){
-     int ra_i = threading::StructureElement(get_model(),a_).get_start_res();
-     int rb_i = threading::StructureElement(get_model(),b_).get_start_res();
+     //int ra_i = threading::StructureElement(get_model(),a_).get_start_res();
+     //int rb_i = threading::StructureElement(get_model(),b_).get_start_res();
```

Comment:
If not needed any more, just delete rather than commenting out. We have the old code in source control so can always get it back if we need it again.